R/authors-at-r.R

```diff
@@ -234,3 +234,41 @@ idesc_get_maintainer <- function(self, private) {
     NA_character_
   }
 }
+
+
```

Comment:
Can you please add documentation?

Reply:
I've added docs in the `generate_api` and the `description` documentation